frontend/packages/metal3-plugin/src/components/dashboard/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { StatusGroupMapper } from '@console/internal/components/dashboard/inventory-card/inventory-item';
+import { InventoryStatusGroup } from '@console/internal/components/dashboard/inventory-card/status-group';
+
+import { getSimpleHostStatus } from '../../utils/host-status';
+import {
+  HOST_ERROR_STATES,
+  HOST_PROGRESS_STATES,
+  HOST_PROVISIONING_STATES,
+  HOST_REGISTERING_STATES,
+  HOST_SUCCESS_STATES,
+} from '../../constants';
+
+const BMH_STATUS_GROUP_MAPPER = {
+  [InventoryStatusGroup.OK]: [...HOST_SUCCESS_STATES],
+  [InventoryStatusGroup.PROGRESS]: [
+    ...HOST_PROGRESS_STATES,
+    ...HOST_PROVISIONING_STATES,
+    ...HOST_REGISTERING_STATES,
+  ],
+  [InventoryStatusGroup.ERROR]: [...HOST_ERROR_STATES],
+};
+
+export const getBMHStatusGroups: StatusGroupMapper = (bmhs) => {
+  const groups = {
+    [InventoryStatusGroup.OK]: {
+      statusIDs: ['ready', 'provisioned'],
```

Comment:
I believe the statusIDs be the host status constants from `../../constants`

Reply:
these statusIDs are keys in this object https://github.com/openshift/console/blob/master/frontend/packages/metal3-plugin/src/components/table-filters.ts#L14 .  Most of them are in constants (missing `other`) but It seems to me that those constants are values which can appear in bmh status K8s object and are not meant to be IDs of status filters. Correct me if Im wrong.

Reply:
You're right, thanks for clarification.